### PR TITLE
Removed create-cert init containers

### PR DESCRIPTION
**Pull request by Bobbins228**
# Issue link

[RHOAIENG-5086](https://issues.redhat.com/browse/RHOAIENG-5086)
# What changes have been made

Removed `create-cert` init containers and related volumes + environment variables.
Remove local_interactive`

# Verification steps

### Notebook server ODH/RHOAI/Local
* Clone this repository with `git clone https://github.com/project-codeflare/codeflare-sdk.git`
* Checkout this PR's branch
* Run `poetry build` - install if needed (`pip install poetry`)
* Run `pip install --force-reinstall dist/codeflare_sdk-0.0.0.dev0-py3-none-any.whl` 
* Restart your notebook kernel
## Testing
* Create a Ray Cluster with `ClusterConfiguration`/`cluster.up()`
* Inspect that the yaml file does not include the `creat-cert` init containers, `RAY_TLS` related env variables, `ca-vol` volumes or the Oauth side car container

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: src/codeflare_sdk.egg-info/PKG-INFO [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Metadata-Version: 2.1
-Name: codeflare-sdk
+Name: codeflare_sdk
```
Comment (ChristianZaccaria) [inline src/codeflare_sdk.egg-info/PKG-INFO:2]: Is this change required?

File: pyproject.toml [hunk 2]
```diff
@@ -48,6 +48,7 @@ pytest-timeout = "2.2.0"
 [tool.pytest.ini_options]
 filterwarnings = [
     "ignore::DeprecationWarning:pkg_resources",
+    "ignore::DeprecationWarning",
```
Comment (ChristianZaccaria) [inline pyproject.toml:51]: I'm not sure if we want to ignore 'all' deprecation warnings.

File: src/codeflare_sdk/utils/generate_yaml.py [hunk 3]
```diff
@@ -620,12 +506,13 @@ def generate_appwrapper(
...
     if local_interactive:
-        enable_local_interactive(resources, cluster_name, namespace)
-    else:
-        disable_raycluster_tls(resources["resources"])
+        enable_local_interactive(resources)
 
-    if is_openshift_cluster():
-        enable_openshift_oauth(user_yaml, cluster_name, namespace)
+    # else:
+    #     disable_raycluster_tls(resources["resources"])
```
Comment (ChristianZaccaria) [inline src/codeflare_sdk/utils/generate_yaml.py:512]: I think these comments can be removed

**Review by ChristianZaccaria — CHANGES_REQUESTED** [review]

File: src/codeflare_sdk.egg-info/PKG-INFO (hunk 1 above)
Comment (Ygnas) [reply src/codeflare_sdk.egg-info/PKG-INFO:2]: Why is the egg-info not in the .gitignore in the first place?

File: pyproject.toml (hunk 2 above)
Comment (Bobbins228) [reply pyproject.toml:51]: Was testing something - will remove

File: src/codeflare_sdk.egg-info/PKG-INFO (hunk 1 above)
Comment (Bobbins228) [reply src/codeflare_sdk.egg-info/PKG-INFO:2]: @Ygnas Very good question!
I smell a handy PR :)

File: src/codeflare_sdk/utils/generate_yaml.py [hunk 4]
```diff
@@ -620,12 +506,13 @@ def generate_appwrapper(
     )
 
     if local_interactive:
-        enable_local_interactive(resources, cluster_name, namespace)
-    else:
-        disable_raycluster_tls(resources["resources"])
+        enable_local_interactive(resources)
```
[the remark this answers is not in the document]
Comment (Bobbins228) [reply src/codeflare_sdk/utils/generate_yaml.py:509]: Will be removed in #512

[the remark this answers is not in the document]
Comment (Bobbins228) [reply src/codeflare_sdk/utils/generate_yaml.py]: Added it to this PR for less mess
